### DISCUSSION TITLE
FreeBox et Starlink enfin ca fonctionne !

### DISCUSSION THREAD

**Discussion by jccourtoy** [Ysurac/openmptcprouter#2726] [Show and tell]
Bonjour,

Je vous partage mon expérience avec TUTO
La Freebox ne fonctionne pas correctement avec OpenMPTC a cause de l'IPv6. ( Toutefois j'ai pas bien compris pourquoi)
Et, le Starlink ne permet pas de désactiver le DHCP

Alors, il faut passer par un petit investissement, j'ai pris deux Adaptateurs Réseau Ethernet (LAN) le moins cher sur Amazon :
https://www.amazon.fr/Mobi-Lock-Adaptateur-P%C3%A9riph%C3%A9riques-Compatibles/dp/B00NLL10P4/ref=sr_1_5?keywords=Mobi+Lock&qid=1673279544&sr=8-5

Configuration : 
Ajouter eth1 avec une configuration DHCP 
![Screenshot from 2023-01-09 16-53-57](https://user-images.githubusercontent.com/78306988/211351250-a92166c1-7fa8-4887-af18-a565c50c3742.png)
Puis ajouter eth2 pour la Freebox dans mon cas j'ai remis la Freebox en DHCP, car le fonctionnement est meilleur.
![Screenshot from 2023-01-09 16-54-13](https://user-images.githubusercontent.com/78306988/211351429-1e8002c2-e892-4e1d-b519-098ebdfdc3b7.png)
Résultat au Top
![Screenshot from 2023-01-09 16-57-59](https://user-images.githubusercontent.com/78306988/211352153-e219e57d-3b96-4ac2-80a1-7b3cec0f712a.png)

J'ai également réglé le mtu de chaque interface
Dans Interface> Appareil
eth1 Starlink le mtu est 1472
eth2 la Freebox le mtu est 1492
trouver sur le web, 
![14183445093](https://user-images.githubusercontent.com/78306988/211353403-a3b8fe56-bafe-4ecb-b1e6-ecd23ac1bbe3.png)

J'espère que cela pourra aider d'autre, j'ai mis beaucoup de temps à trouver les causes, avec un speed test qui ne dépassait jamais les 8Mbps avec une box 5g.

**Comment by haribeau74** [comment]
Bonjour

je suis dans la même config que toi,
Si je comprends bien tu branches en usb 3 ton routeur Starlink sur le rasperry,
Mais pourquoi le second adaptateur  usb ethernet tu branches ta Freebox aussi dessus ?

Tu diffuses le wifi, et l ethernet par quel matériel ensuite ta Freebox ou tu as un switch ?

merci

**Comment by jccourtoy** [comment]
Le WiFi est diffusé par des modules Tenda qui sont interconnectés en Mesh, lesquels sont vendus par Amazon. J'ai également connecté ma Freebox en USB, car sinon elle est limitée à une vitesse de 7 mégabits par seconde. Si j'ai bien compris, cette limitation est due à l'utilisation d'IPv6 que la Freebox nécessite d'injecter. Je ne suis pas sûr de comprendre la raison exacte de cette limitation, mais je sais que cela a un rapport avec IPv6. En tout cas, c'est ce que j'ai compris.

**Reply by haribeau74** [reply]
Merci pour ton retour,
Pour ma part, j ai désactivé l ip6 sur ma Freebox,
Je suis en attente de l adaptateur ethernet/usb

Tu avais mis ton Starlink en mode contournement wifi ?


avec ma config actuelle
Freebox sur lequel le Starlink et rasperry connectée en ethernet 
Si je passe mon Starlink en mode contourmenent, il ne fonctionne plus.


Bonne soirée

**Comment by jccourtoy** [comment]
Ben non rien de tout cela, mon sterling est connecté en ethernet USB sur le raspberry et la Freebox. Pareil. Sur ma freebox, aucune manipulation de désactivation de IPv6. L'activation via l'espace client ne fonctionne pas.
Tu pourrais-tu pour résumer l'Internet du raspberry va sur mon Wi-Fi, et deux entrées en ethernet USB pour la Freebox et le sterling

**Comment by haribeau74** [comment]
Ok
Donc ton rasperry tu le branches en ethernet sur ton Switch/ wifi qui diffuse ensuite 
Et ton Starlink tu as mis en mode contournement routeur wifi ou pas ?


Désolé pour toute ces questions,
Mais j aimerai bien finaliser mon installation

**Reply by jccourtoy** [reply]
Non je l'ai branché sur un USB du rpi avec le module ethernet USB
Pareil avec la Freebox

**Comment by Brotiss84** [comment]
Bonjour, 

J'envisageais d'installer Starlink tout en conservant ma box delta (FreeBox).
La solution a l'air top. J'ai besoin de conseils concernant le système de configuration et l'application que vous utilisiez.
Par avance Merci de votre aide.

**Reply by jccourtoy** [reply]
Parfait suit le tuto.
D'abord installer la solution openmptcrouter puis suivre la config.
